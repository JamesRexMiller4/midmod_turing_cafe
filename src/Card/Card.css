.card-wrapper-div {
  margin-left: 60px;
  margin-bottom: 20px;
  width: 70%;
  background-color: white;
  border: 5px solid;
  border-color: teal;
  border-radius: 5%;
}