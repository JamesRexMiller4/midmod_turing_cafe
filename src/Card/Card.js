const Card = ({ name, date, time, guests}) => {
  return ( 
    <div>
      <h2>{name}</h2>
      <h3>{date}</h3>
      <h3>{time}</h3>
      <h3>Number of Guests: {guests}</h3>
    </div>
   );
}
 
export default Card;