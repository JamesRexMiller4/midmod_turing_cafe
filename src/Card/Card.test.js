import React from 'react';
import {shallow} from 'enzyme';
import Card from './Card.js'
 
describe('Card', () => {
let wrapper;
 
beforeEach(() => {

wrapper = shallow(<Card
  name={'Jerry'}
  time={'12:12'}
  date={'12/12'}
  number={4}
/>)
})

 
it('should match the snapshot with all the data passed through', () => {
  expect(wrapper).toMatchSnapshot()
})
 
});