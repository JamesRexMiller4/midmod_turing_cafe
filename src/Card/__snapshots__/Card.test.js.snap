// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match the snapshot with all the data passed through 1`] = `
<div
  className="card-wrapper-div"
>
  <h2>
    Jerry
  </h2>
  <h3>
    12/12
  </h3>
  <h3>
    12:12
  </h3>
  <h3>
    Number of Guests: 
    4
  </h3>
  <button
    className="cancel btn"
  >
    Cancel
  </button>
</div>
`;
